src/register/data/tests/reducers.test.js

```diff
@@ -79,7 +111,68 @@ describe('Registration Reducer Tests', () => {
       },
     );
   });
+  it('should set the register user when pipline data is loaded', () => {
+    const payload = { value: true };
+    const action = {
+      type: REGISTER_SET_USER_PIPELINE_DATA_LOADED,
+      payload,
+    };
+
+    expect(reducer(defaultState, action)).toEqual(
+      {
+        ...defaultState,
+        userPipelineDataLoaded: true,
+      },
+    );
+  });
+
+  it('dont set country code if this is already setup', () => {
```

Comment:
The test logic is setting the country but the test is don't set the country code. could you explain it?

Reply:
In this we are setting the country code because that is already setup so we will skip the if condition which is written in reducer

Reply:
addressed